src/js/shared/SkeletonUI.js

```diff
@@ -0,0 +1,34 @@
+import { $createElement } from "../utils/dom";
+
+class SkeletonUI {
+  constructor(count) {
+    const $ul = $createElement("ul", this.renderSkeleton(count));
+    $ul.classList.add("skeleton-cards", "item-list", "absolute");
+    this.$ul = $ul;
+  }
+
+  renderSkeleton(count) {
+    const skeletonCard = `
+      <li>
+        <a href="#"> 
+          <div class="item-card"> 
+            <div class="item-thumbnail skeleton"></div> 
+            <div class="item-title skeleton"></div> 
+            <div class="item-score skeleton"></div> 
+          </div> 
+        </a> 
+      </li>`.repeat(count);
+
+    return skeletonCard;
+  }
+
+  showSkeleton() {
+    this.$ul.classList.remove("hidden");
+  }
+
+  hideSkeleton() {
+    this.$ul.classList.add("hidden");
+  }
```

Comment:
```suggestion
  show() {
    this.$ul.classList.remove("hidden");
  }

  hide() {
    this.$ul.classList.add("hidden");
  }
```
어차피 자기자신에 대한 조작이라서 이렇게만 표현해줘도 무방해보여요!